cob_bringup/drivers/sick_lms1xx.launch

```diff
@@ -6,11 +6,11 @@
 	<arg name="name"/>
 
 	<!-- load parameter -->
-	<rosparam ns="$(arg name)" command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg name).yaml"/>
+	<rosparam ns="$(arg name)/driver" command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg name).yaml"/>
```

Comment:
that probably needs modification in the implementation of cob_driver/cob_sick_lms1xx package? @destogl?